.github/workflows/CI-CD.yml

```diff
@@ -0,0 +1,40 @@
+name: CI-CD
+
+on: push
+
+jobs:
+  build:
+    name: Lint, build and test
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Setup yarn cache
+        uses: c-hive/gha-yarn-cache@v2
+      - name: Setup NodeJS 14
+        uses: actions/setup-node@v2
+        with:
+          node-version: '14'
+      - run: yarn install
+      - run: yarn lint
+      - run: yarn build
+      - run: yarn test
+  deploy:
+    name: Deploy on github pages
+    if: contains(github.ref, 'master')
```

Comment:
Why not check straight for equality? `${{ github.ref == 'master' }}` 
Do we want to deploy pages on `some-master-branch` kind of branches as well?

Reply:
True, however it doesn't work that way. It should be `github.ref == 'refs/heads/master'`